### PR TITLE
removed checksum and fingerprint. updated tests.

### PR DESCRIPTION
**Pull request by visualphoenix**
checksum and fingerprint seem to cause 500 response errors. update tests.

### CODE REVIEW

**Comment by visualphoenix** [comment]
so i'm not sure how to update the mocked responses properly. I included a real response. If i removed the mocks and actually made calls to sentry using their newer event api i was able to get those responses.

**Comment by visualphoenix** [comment]
the biggest change here is now we don't lose the tag from the event if we choose to add extra tags

File: test/plugin/test_out_sentry.rb [hunk 1]
```diff
@@ -42,28 +43,45 @@ def test_emit
...
         'message' => emit_message,
-        'something' => emit_extra
+        'something' => emit_extra,
+        'tags' => {'cool' => 'bar'},
+        'timestamp' => emit_timestamp,
+        'logger' => emit_logger,
+        'server_name' => emit_server_name,
+        'release' => emit_release,
+        'time_spent' => emit_time_spent,
+        'culprit' => emit_culprit
```
Comment (y-ken) [inline test/plugin/test_out_sentry.rb]: Would you please add another method to run tests?

* move stub_request to another method
* add `test_emit_mock` like `test_emit`

Comment (visualphoenix) [inline test/plugin/test_out_sentry.rb]: so to clarify, i'll 
* rename `stub_endpoint` to `stub_post`
* create a new `stub_response`
* revert `test_emit` back to the old tests but call `stub_post` now
* add a new `test_emit_mock` method
* make `test_emit_mock` similar to the current test

Comment (y-ken) [inline test/plugin/test_out_sentry.rb]: Thanks, Good job!!

**Comment by visualphoenix** [comment]
happy to help.

**Pull request merged**